src/constants.ts

```diff
@@ -44,3 +44,15 @@ export const UDC = {
   ADDRESS: '0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf',
   ENTRYPOINT: 'deployContract',
 };
+
+export const RPC_GOERLI_NODES = [
```

Comment:
Maybe we shouldn't include the `/rpc/v0.5` part in the string and have only the base URL there. Then, in `"RpcProviderOptions"`, we could allow clients to specify the expected RPC version. This would make our API future-proof, assuming these endpoints will also be relevant for v0.6 (and beyond?)

Reply:
We should probably request Juno to maintain path compatibility with other nodes, for example, `/rpc/v0.X` instead of `/v0_X`. Then snjs could simply append the expected version (or use the default if no version is specified) based on user-provided options.

Reply:
Regarding /rpc/v0.X it was my initial idea also, but it looks like it will not be possible to sync all services.
But this pulls also another question, at the moment with the latest version we support only the latest RPC Spec. To change that we would need to:
Case1 -  Maintain in parallel multiple versions each one related to specific RPC.
Case 2 - Support Two RPC Versions as the same codebase until the next one comes, then remove first and add the latest.

Case 2 is what you are proposing and this could be done when we:
1. Remove Sequencer Provider (it is now only deprecated)
2. Remove the common interface Between RPC and Sequencer (my plan was to first deprecate it, then let it exist for some time after the removal of Sequencer provider for the migration period, then remove it completely)
3. Move the default provider from using a common interface to using the RPC Provider interface
4. And then we have space to introduce two RPC Provider Default implementations one for 0.N-1 and another for 0.N RPC Specification.